tools/clang/lib/SPIRV/SpirvBuilder.cpp

```diff
@@ -882,15 +875,77 @@ void SpirvBuilder::createBarrier(spv::Scope memoryScope,
   insertPoint->addInstruction(barrier);
 }
 
-SpirvBitFieldInsert *SpirvBuilder::createBitFieldInsert(
-    QualType resultType, SpirvInstruction *base, SpirvInstruction *insert,
-    SpirvInstruction *offset, SpirvInstruction *count, SourceLocation loc) {
+SpirvInstruction *SpirvBuilder::createEmulatedBitFieldInsert(
+    QualType ResultType, uint32_t BaseTypeBitwidth, SpirvInstruction *Dst,
+    SpirvInstruction *Src, unsigned BitOffset, unsigned BitCount,
+    SourceLocation loc, SourceRange range) {
+
+  assert(BitCount <= 64 &&
+         "Bitfield insertion emulation can only insert at most 64 bits.");
+  auto *DstTy = dyn_cast<IntegerType>(Dst->getResultType());
```

Comment:
I know it's inconsistent with the LLVM style guide, but to be consistent with the rest of this file and the SPIR-V backend, can you switch to lowerCamelCase variable names?